dashboard/app/views/levels/editors/_dsl.html.haml

```diff
@@ -25,23 +25,7 @@
 .script_text
   = text_area_tag('level[dsl_text]', @level.dsl_text || (@level.dsl_default))
 
-- if @level.is_a?(External)
-  .field
-    = f.label :icon_type, 'Icon'
-    %p
-      Override the icon that shows up for this level
-      %div
-        Default:
-        %i.fa.fa-file-text
-        Stop:
-        %i.fa.fa-stop-circle
-        Map:
-        %i.fa.fa-map
-
-    = f.select :icon_type, options_for_select(["Default", "Stop", "Map"], @level.icon_type)
```

Comment:
Turns out the icon stuff for stop levels wasn't working correctly, so removing that for now.